sky/shell/BUILD.gn

```diff
@@ -221,6 +221,8 @@ if (is_android) {
 
     extensions_to_not_compress = ".flx"
 
+    flutter_dist_jar = "$root_build_dir/Flutter.jar"
```

Comment:
Is it common style to use a capital letter at the start of a jar name?  The handful of examples I'm familiar with have all be lowercase names.